### DISCUSSION TITLE
Is there a way to update from 0.8.12 to the current or do I need to do a fresh install?

### DISCUSSION THREAD

**Discussion by frrobert2** [datenstrom/community#820] [Ask a question] [answered]
I tried updating my website from 0.8.12 using the update command but I get the following error:

ERROR updating files: Can't download file https://github.com/datenstrom/yellow-extensions/raw/master/version.ini

Is there a way to update from 0.8.12 to the current or do I need to do a fresh install?

Thanks

**Comment by schulle4u** [comment, marked as answer]
Sorry, the oldest version from wich you can update to the most recent version is 0.8.15. Please do a fresh install and re-install/modify the desired extensions and content files. Let us know if you need help during the process.

**Reply by frrobert2** [reply]
Thank you

**Comment by annaesvensson** [comment]
Tip: You can let us know when your question has been answered, click on `Mark as answer`.
